app/javascript/components/components/Header.js

```diff
@@ -0,0 +1,53 @@
+import React, { Component } from 'react'
+import {
+  Navbar,
+  NavItem,
+  NavLink,
+  NavbarToggler,
+  NavbarBrand,
+  Collapse,
+  Nav,
+  NavbarText} from 'reactstrap';
+
+class Home extends Component {
```

Comment:
This file is called Header but the class is called Home. That feels a little questionable.